integr8ly/integreatly-operator
# MGDAPI-2160 - adding update to config map for redis service updates + CRO bump to v0.28.0

# Description
JIRA: https://issues.redhat.com/browse/MGDAPI-2160
Adding first steps to update the cloud-resources-aws-strategies config map redis type with a serviceUpdates entry where a list of strings can be added which cro would then apply.
Integreatly-operator will ad a list f hard-coded service versions to CRO config map. It should also preserve versions added manually.

## Verification steps
- Install RHOAM onto a cluster with clusterStorage = false
- Check the cloud-resources-aws-strategies config map in redhat-rhoam-operator NS - redis config for production should contain:
`"serviceUpdates": ["elasticache-20210615-002"]`
- Verify that CRO v0.28.0 was installed
- Verify that CRO pod doesn't crash (0 restarts)
- Check logs of the CRO pod - there should not be any errors related to Redis provisioning or reconciliation

_Description updated by @matskiv on Fri July 2nd._

## Type of change




## Checklist
